site/src/components/advice-callout/AdviceCallout.module.css

```diff
@@ -0,0 +1,30 @@
+.root {
+  padding: calc(2 * var(--salt-size-unit)); /* var(--salt-size-container-spacing); */
```

Comment:
Good point.

The 2x value looks closer to what we did on the UITK site though. If it's Ok with you, I'll leave as is but make a note to query this with @shey-v when we sit down to review and tweak the styling for this component.

As it happens, `--salt-size-container-spacing` didn't seem to be available anyway (possibly because `foundations-docs` branched off `main` a while ago and needs a catch-up).